spec/features/observer_spec.rb

```diff
@@ -3,7 +3,7 @@
     expect(CommunicartMailer).to receive_message_chain(:on_observer_added, :deliver_later)
 
     work_order = create(:ncr_work_order)
-    observer = create(:user)
+    observer = create(:user, client_slug: 'ncr')
```

Comment:
maybe we should create a factory called `ncr_user` or add `ncr` as a trait on `user` that sets the client_slug so we don't have to set this in various tests?